app/Mail/NoPaper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class NoPaper extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $userName;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param string $userName
+     */
+    public function __construct(string $userName)
+    {
+        //
+        $this->userName = $userName;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->markdown('emails.no_paper')
+            ->subject('No paper in printer');
```

Comment:
You should use the `__()` function here, this work like the `@lang` blade directive.